jsbridge/src/main/kotlin/de/prosiebensat1digital/oasisjsbridge/extensions/XMLHttpRequestExtension.kt

```diff
@@ -175,13 +175,66 @@ internal class XMLHttpRequestExtension(
  * limitations under the License.
  */
 const val xhrJsCode: String = """
-var XMLHttpRequest = function() {
-  this._send_native = XMLHttpRequestExtension_send_native;
+(function() {
+  var sendNative = XMLHttpRequestExtension_send_native;
+
+function isListenerWithMathingName(eventName) {
```

Comment:
Looks like a typo

Reply:
Absolutely. Thanks, should be fixed now.